packages/commonwealth/client/scripts/views/components/MobileNavigation/MobileNavigation.tsx

```diff
@@ -75,10 +76,25 @@ const MobileNavigation = () => {
   const isDiscussionsPage = window.location?.pathname?.includes('/discussion');
   const shouldHideQuickPost = isNewThreadPage || isDiscussionsPage;
 
+  useEffect(() => {
+    const node = navigationRef.current;
+    if (!node) return;
+
+    const preventScroll = (event: TouchEvent) => {
+      event.preventDefault();
+    };
+
+    node.addEventListener('touchmove', preventScroll, { passive: false });
```

Comment:
Does passive false lock up the UI? if not them I'm good with it... 

passive event listeners improve UI performance normally.

Reply:
from my CA it seemed totally fine (local device testing) / simulated normal usage